core/block/export/export.go

```diff
@@ -175,6 +183,27 @@ func (e *export) Export(ctx context.Context, req pb.RpcObjectListExportRequest)
 	return wr.Path(), succeed, nil
 }
 
+func (e *export) sendNotification(err error, req pb.RpcObjectListExportRequest) {
+	errCode := model.NotificationExport_NULL
+	if err != nil {
+		errCode = model.NotificationExport_UNKNOWN_ERROR
+	}
+	notificationSendErr := e.notificationService.CreateAndSendLocal(&model.Notification{
+		Id:         uuid.New().String(),
```

Comment:
seems like setting of id and create time could be encapsulated inside notification service